### PR TITLE
Career quiz

### PR DESCRIPTION
**Pull request by ccshah298**
# Description

Created Carrer Quiz Page Component

Fixes # (15)

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/containers/QuizPage/QuizPage.component.js [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+      <div className="middle_page">
+        <div className="sidemenu_page">
+          <SideMenu highLigtItem={3} />
+        </div>
+        <div className="question_page">
+          <QuestionComponent
+            imageSrc={image1}
+            question="When visiting a website, what is that you are most interested in?"
+          />
+          <div className="quizans_page">
```
Comment (sofiiadidovych) [inline src/client/containers/QuizPage/QuizPage.component.js]: Charmi, I didn`t find this class in your css file, so maybe it should be deleted.

**Review by sofiiadidovych — APPROVED** [review]
Great job! Your storybook looks nice, and the code is clean and clear.

File: src/client/containers/QuizPage/QuizPage.component.js (hunk 1 above)
Comment (ccshah298) [reply src/client/containers/QuizPage/QuizPage.component.js]: Thank you so much @sofiiadidovych for review my PR.. i will do.. :)

**Review by sandhana14 — APPROVED** [review]
I think so, Everything looks perfect

File: src/client/containers/QuizPage/QuizPage.component.js [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+            <SideMenu highLigtItem={3} />
+          </div>
+          <div className="question_page">
+            <QuestionComponent
+              imageSrc={image1}
+              question="When visiting a website, what is that you are most interested in?"
+            />
+            <div className="quizans_page">
+              <QuizAnswers
+                isAgreementQuestion={false}
```
Comment (rajanibusani) [inline src/client/containers/QuizPage/QuizPage.component.js:29]: Hi Charmi, If i choose isAgreementQuestion={true} here, it's showing agree or disagree options, but it looks like they require some css to align them properly. Have a look Once...

**Review by rajanibusani — COMMENTED** [review]
Hi Charmi...U really did a very good job with quiz page..👏

File: src/client/containers/QuizPage/QuizPage.component.js (hunk 2 above)
Comment (ccshah298) [reply src/client/containers/QuizPage/QuizPage.component.js:29]: Thanks @rajanibusani for reviewing my PR.. I tried AgreementQuestion={true}  but i think for now as per requirement this one looks ok..  Thanks again :)

Comment (ccshah298) [reply src/client/containers/QuizPage/QuizPage.component.js:29]: Thanks @sandhana14 for review my PR :)

**Review by rajanibusani — APPROVED** [review]

File: src/client/containers/QuizPage/QuizPage.component.js [hunk 3]
```diff
@@ -1,3 +1,45 @@
...
+export const QuizPage = () => {
+  return (
+    <div>
+      <div className="main">
+        <div className="header_page">
+          <PageHeader />
+        </div>
+        <div className="middle_page">
+          <div className="sidemenu_page">
+            <SideMenu highLigtItem={3} />
```
Comment (bytewiz) [inline src/client/containers/QuizPage/QuizPage.component.js:20]: I think there is a typo here `highLightItem={3}` missing an `h`

**Review by bytewiz — CHANGES_REQUESTED** [review]

File: src/client/containers/QuizPage/QuizPage.component.js (hunk 3 above)
Comment (ccshah298) [reply src/client/containers/QuizPage/QuizPage.component.js:20]: Thanks for review.. done

**Review by bytewiz — APPROVED** [review]

**Pull request merged**